src/main/views/lists/tabs/contactTypes/editContactType.njk

```diff
@@ -0,0 +1,71 @@
+{% from "govuk/components/input/macro.njk" import govukInput %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+
+{% set isNewContactType = true %}
+{% if contactType.id %}
+  {% set isNewContactType = false %}
+{% endif %}
+
+{% if isNewContactType %}
+  <h2 class="govuk-heading-l">Add New Contact Type</h2>
+{% else %}
+  <h2 class="govuk-heading-l">Editing Contact Type: {{contactType.type}}</h2>
+{% endif %}
+
+{% if errors|length %}
+  {{ govukErrorSummary({
+    titleText: "There is a problem",
+    errorList: errors
+  }) }}
+{% endif %}
+
+{% if fatalError == false %}
```

Comment:
Double checking. Guessing if there is a fatal error, it appears in the list above?

Reply:
yes all errors will appear in list